scripts/remote/README.md

```diff
@@ -62,7 +78,8 @@ e.g. `host` -> `PASH_REMOTE_HOST`, or `my-conf1g-key` -> `PASH_REMOTE_MY_CONF1G_
 
 ### API
 
-* `/run` runs `~/work.sh`.
+* `/run` runs the worker's `~/work.sh`, with added invariants
+  depending on the server's [command module][cm].
```

Comment:
Is `[cm]` intended to be `(cm)` (if for Markdown link?) . 

(You can also link directly to the subsection title using `(#command-modules)` :)

Reply:
This is valid syntax for [full-reference style links](https://github.github.com/gfm/#full-reference-link). See line 116 for the named reference to `#command-modules`.

Reply:
Thanks for this!